jenkins/catapult/jenkins/catapult-client-build-base-image-all.groovy

```diff
@@ -142,21 +148,55 @@ pipeline {
 			}
 		}
 	}
+	post {
+		success {
+			script {
+				if (env.SHOULD_PUBLISH_JOB_STATUS?.toBoolean()) {
+					helper.sendDiscordNotification(
+							':confetti2: Catapult Client All Image Job Successfully completed',
```

Comment:
notice that these (and some other) arguments are double indented but others are only single indented

probably better to be consistent

Reply:
yea thanks, VS Code 😱